Community-Committee-Charter.md

```diff
@@ -0,0 +1,128 @@
+
+## Section 9. Roles
+
+Node.js Foundation git repositories for Community Committee Initiatives
+are maintained by the CC and additional Collaborators who are added by
+the CC on an ongoing basis.
+
+Individuals making significant and valuable contributions,
+“Contributor(s)”, are made Collaborators and given commit-access to the
+project. These individuals are identified by the CC and their addition
+as Collaborators is discussed during the weekly CC meeting.
+Modifications of the contents of the git repository are made on a
+collaborative basis as defined in the development process.
+
+Collaborators may opt to elevate significant or controversial
+modifications, or modifications that have not found consensus to the CC
+for discussion by assigning the `cc-agenda` tag to a pull request or
+issue. The CC should serve as the final arbiter where required. The CC
+will maintain and publish a list of current Collaborators by Project, as
+well as a development process guide for Collaborators and Contributors
+looking to participate in the development effort.
+
+Individual Membership Directors of the Node.js Foundation are the Node.js
+project’s community voice on the board. There are two individual directors
+that sit on the Node.js Foundation board and they each serve a two-year term.
+Each Individual Membership Director is responsible for soliciting feedback and
+data that represents the wishes of other individual members and the community
+at large. These directors participate and observe CC meetings in order to deliver
+monthly updates to the Board of the project. They deliver monthly updates to the
+CC on what the Board is doing. They have been entrusted with the duty to make
+decisions based on the information they receive to best represent the community,
+and can gather input for proposals when relevant and granted permission to do
+so by the Board.
+
+## Section 10. Definitions
+
+- Contributors: contribute code or other artifacts, but do not have the right to commit to the code base. Contributors work with the Project’s Collaborators to have code committed to the code base. A Contributor may be promoted to a Collaborator by the projects’ Maintainer or the CC. Contributors should rarely be encumbered by the CC and never by the Board.
+- Project: a collaboration effort, e.g. a subsystem, that is organized through the project creation process and approved by the CC.
+- Community Project: projects within the Node.js Foundation or in the ecosystem that contribute to the health of the Node.js project.
+- Adviser: a Collaborator within a Community Project elected to represent the Community Project on the CC.
+- Member: a person who participates in the development of the Community Committee through code or other artifacts.
```

Comment:
I like these definitions, but I fear the last one may be confusing with the Member definition from CONTRIBUTING.md and GOVERNANCE.md

Reply:
This is what currently exists in the charter and requires board approval to change. I would advise keeping changes scoped to the Director for this upcoming board meeting.

Reply:
In that case, we would need to get a second approval for this change if I'm not mistaken?

Anyway, if you think that making both changes at once would risk being detrimental to the overall approval of this PR, I'd say we stick to your advice. the priority would be to get the main changes done here approved.